### DISCUSSION TITLE
A fixed 3d model node

### DISCUSSION THREAD

**Discussion by Arun1947** [sceneview/sceneview#90] [Q&A]
How to create a fixed 3d model node in SceneView (for example, to make the model visible on the screen even if the camera is moved).

**Comment by grassydragon** [comment]
You can add a node as a child of the camera.

**Comment by ThomasGorisse** [comment]
In v4.0.0-rc.1, place a fixed (non-draggable) model in AR by anchoring it:

```kotlin
ARSceneView {
    AnchorNode(anchor = anchor) {
        ModelNode(
            modelInstance = model,
            scaleToUnits = 1.0f
        )
    }
}
```

Anchors are fixed in world space by default — the model stays in place as the user moves around. For a 3D-only scene without AR, just use `SceneView` with a fixed `position` on your `ModelNode`.
